scripts/test_psa_compliance.py

```diff
@@ -141,9 +141,18 @@ def main(library_build_dir: str):
     parser = argparse.ArgumentParser()
     parser.add_argument('--build-dir', nargs=1,
                         help='path to Mbed TLS / TF-PSA-Crypto build directory')
+    parser.add_argument('--expected-failures', nargs='+',
+                        help='''set the list of test codes which  are expected to fail
```

Comment:
```suggestion
                        help='''set the list of test codes which are expected to fail
```
Incredibly small nit but double space here.